lib/event.js

```diff
@@ -687,6 +702,15 @@ var ICalEvent = function(_data, calendar) {
                 g += ';BYMONTHDAY=' + data.repeating.byMonthDay.join(',');
             }
 
+            if(data.repeating.exclude) {
+                g += ';EXDATE=';
```

Comment:
Good catch, I'll write a test for it this weekend after I get some work done and fix it.